notebooks/tel_and_site/subsys_req_ver/m1m3/SITCOM-1136_M1M3_analysis_stability_tracking.ipynb

```diff
@@ -0,0 +1,704 @@
+{
```

Comment:
<p>Please, add some descript of what you are going to do through this section. Also, in the subsections, please, add a few comments on the purpose of each cell, when not obvious to the reader point of view.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-sitcom/notebooks_vandv/pull/107/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Reply:
<p>I have included some comments. I hope it is now clearer</p>

Reply:
<p>I cannot see any commit with the changes, so I wond if have pushed your local changes to the remote repository?</p>

Reply:
<p>But I could not see the comments you added even by cloning your version to my nublado. I do see some section you created, but not comments. </p>

Reply:
<p>I don't know what could have happened. But I have added a new cell with comments. Please let me know if you see them or if there is a problem.</p>